test/post_tagging_test.exs

```diff
@@ -0,0 +1,62 @@
+defmodule Bonfire.Tag.PostsTest do
```

Comment:
I'd wrap the entire module in `if Code.ensure_loaded?(Bonfire.Posts) do` so it only compiles and runs if the posts extension is available